message_ix/model/MESSAGE/model_core.gms

```diff
@@ -946,7 +955,13 @@ ACTIVITY_BOUND_UP(node,tec,year,mode,time)$( map_tec_act(node,tec,year,mode,time
 * unless explicitly stated otherwise.
 ***
 ACTIVITY_BOUND_LO(node,tec,year,mode,time)$( map_tec_act(node,tec,year,mode,time) )..
```

Comment:
please remove `$( map_tec_act(node,tec,year,mode,time) )`, otherwise this will not work for lower bounds on `mode='all'`